qiskit/assembler/__init__.py

```diff
@@ -15,25 +15,19 @@
 Circuit and Schedule Assembler (:mod:`qiskit.assembler`)
 ========================================================
 
-.. currentmodule:: qiskit.assembler
+Functions
+=========
 
-Circuit Assembler
-=================
```

Comment:
These headers each only had one entry. It makes the page a bit noisy: https://docs.quantum.ibm.com/api/qiskit/dev/assembler